### PR TITLE
adding drone-github-changelog-generator plugin

### PR DESCRIPTION
**Pull request by naorlivne**
This pull request adds a plugin version of https://github.com/github-changelog-generator/github-changelog-generator

### CODE REVIEW

File: content/naorlivne/drone-github-changelog-generator/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  image: naorlivne/drone-github-changelog-generator
+  settings:
+    github_user: naorlivne
+    github_project: drone-github-changelog-generator
+    output_path: CHANGELOG.md
+```
+
+## Parameter Reference
+
+#### github_user
```
Comment (tboerger) [inline content/naorlivne/drone-github-changelog-generator/index.md]: Please follow the same notation as the other plugins.

Comment (naorlivne) [reply content/naorlivne/drone-github-changelog-generator/index.md]: Updated.

File: content/naorlivne/drone-github-changelog-generator/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  settings:
+    github_user: naorlivne
+    github_project: drone-github-changelog-generator
+    output_path: CHANGELOG.md
+```
+
+# Parameter Reference
+
+github_user
+
```
Comment (tboerger) [inline content/naorlivne/drone-github-changelog-generator/index.md]: Drop this additional empty line for all flags please.

Comment (naorlivne) [reply content/naorlivne/drone-github-changelog-generator/index.md]: changed.

**Comment by naorlivne** [comment]
The build\publish seemed to fail with the following error: 
`Error response from daemon: network 0_8709931472174675675_default not found`
Which leads me to believe that it was an underlying host issue that prevented the deployment from happening

**Pull request merged**